### ISSUE TITLE
Helper method for checking if using PHPCS v3

### ISSUE THREAD

**Issue by GaryJones** [WordPress-Coding-Standards/WordPress-Coding-Standards#1057]
Once in `develop`, and at least once in the currently open PRs, I see the following:

```php
if ( class_exists( '\PHP_CodeSniffer\Config' ) ) {
	// PHPCS 3.x.
```

The comment could be removed, and the implementation of checking PHPCS version could be centralised if a helper method was introduced - something along the lines of:

```php
public static function is_phpcs3() {
	return class_exists( '\PHP_CodeSniffer\Config' );
}
```

The original code then becomes more self-documenting:

```php
if ( PHPCSHelper::is_phpcs3() ) {
```

If PHPCS 3 should ever remove that `Config` class, or introduces a variable / constant that defines the major version, then the implementation can be switched in one place. Finding and removing the method when PHPCS 2.9 support is dropped may also be a little bit easier.

**Comment by jrfnl** [comment]
This  type of code is not found in any of the sniffs. All these type of checks have already been centralized in the `PHPCSHelper.php` class file (including the one in the open PR you refer to).
That file includes a - currently unused - `PHPCSHelper::get_version()` helper method for when the specific version of PHPCS would be needed somewhere.

As for the specific checks you refer to: as a general rule of thumb, it is better to check for what you **_need_**, i.e. what you will actually use, than to check for a version, so that's how these helper methods have been written.
